core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Workset.java

```diff
@@ -0,0 +1,194 @@
+      @JoinColumn(name = "libraryId") })
+  private Set<Library> libraries;
+
+  @ManyToMany(targetEntity = LibraryDilution.class)
+  @JoinTable(name = "Workset_Dilution", joinColumns = { @JoinColumn(name = "worksetId") }, inverseJoinColumns = {
+      @JoinColumn(name = "dilutionId") })
+  private Set<LibraryDilution> dilutions;
+
+  @ManyToOne(targetEntity = UserImpl.class)
+  @JoinColumn(name = "creator", nullable = false, updatable = false)
+  private User creator;
+
+  @Column(name = "created", nullable = false, updatable = false)
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date creationTime;
+
+  @ManyToOne(targetEntity = UserImpl.class)
+  @JoinColumn(name = "lastModifier", nullable = false)
+  private User lastModifier;
+
+  @Column(nullable = false)
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date lastModified;
+
+  @Override
+  public long getId() {
+    return id;
+  }
+
+  public void setId(long id) {
+    this.id = id;
+  }
+
+  @Override
+  public String getAlias() {
+    return alias;
+  }
+
+  public void setAlias(String alias) {
+    this.alias = alias;
+  }
+
+  public String getDescription() {
+    return description;
+  }
+
+  public void setDescription(String description) {
+    this.description = description;
+  }
+
+  public Set<Sample> getSamples() {
+    if (samples == null) {
+      samples = new HashSet<>();
+    }
+    return samples;
+  }
+
+  public void setSamples(Set<Sample> samples) {
+    this.samples = samples;
+  }
+
+  public Set<Library> getLibraries() {
+    if (libraries == null) {
+      libraries = new HashSet<>();
+    }
+    return libraries;
+  }
+
+  public void setLibraries(Set<Library> libraries) {
+    this.libraries = libraries;
+  }
+
+  public Set<LibraryDilution> getDilutions() {
+    if (dilutions == null) {
+      dilutions = new HashSet<>();
+    }
+    return dilutions;
+  }
+
+  public void setDilutions(Set<LibraryDilution> dilutions) {
+    this.dilutions = dilutions;
+  }
+
+  @Override
+  public User getCreator() {
+    return creator;
+  }
+
+  @Override
+  public void setCreator(User creator) {
+    this.creator = creator;
+  }
+
+  @Override
+  public Date getCreationTime() {
+    return creationTime;
+  }
+
+  @Override
+  public void setCreationTime(Date creationTime) {
+    this.creationTime = creationTime;
+  }
+
+  @Override
+  public User getLastModifier() {
+    return lastModifier;
+  }
+
+  @Override
+  public void setLastModifier(User lastModifier) {
+    this.lastModifier = lastModifier;
+  }
+
+  @Override
+  public Date getLastModified() {
+    return lastModified;
+  }
+
+  @Override
+  public void setLastModified(Date lastModified) {
+    this.lastModified = lastModified;
+  }
+
+  @Override
+  public boolean isSaved() {
+    return getId() != UNSAVED_ID;
+  }
+
+  @Override
+  public String getDeleteType() {
+    return "Workset";
+  }
+
+  @Override
+  public String getDeleteDescription() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getAlias" on line 85. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)